style.css

```diff
@@ -97,3 +101,232 @@
   color: rgba(80, 95, 121, 1);
   font-weight: 900;
 }
+
+.second-section {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-column: auto;
+}
+
+.proj-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding: 4px 0;
+}
+
+.p-descrip {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin: 18px 0;
+}
+
+.lan {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin: 2px;
+}
+
+.proj-lan {
+  display: flex;
+}
+
+.section-2 {
+  background-color: #e5e5e5;
+  padding-bottom: 100px;
+}
```

Comment:
_**THIS COMMENT IS MARKED [OPTIONAL] BECAUSE OF THE AMBIGUOUS INSTRUCTION OF THE PREVIOUS REVIEWER BUT I HIGHLY ADVISE YOU TO MAKE THE CHANGES AS THEY'RE PROJECT REQUIREMENTS**_
- [ ] [OPTIONAL] I'd like to apologize as the previous reviewer highlighted the wrong class name (`second-sec-div`) on the previous comment but the container of your projects card is in fact the div with a `.section-2` class name as I've highlighted above. As per the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) I highly advise you to give it a `grid` display.
- Here's the requirement as a reminder:
![image](https://user-images.githubusercontent.com/104968236/229279519-e33c02ce-52b6-4a9c-b5eb-05be7754741e.png)